homeassistant/components/homeassistant/translations/en.json

```diff
@@ -1,20 +1,21 @@
 {
-    "system_health": {
-        "info": {
-            "arch": "CPU Architecture",
-            "chassis": "Chassis",
-            "dev": "Development",
-            "docker": "Docker",
-            "docker_version": "Docker",
-            "host_os": "Home Assistant OS",
-            "installation_type": "Installation Type",
-            "os_name": "Operating System Family",
-            "os_version": "Operating System Version",
-            "python_version": "Python Version",
-            "supervisor": "Supervisor",
-            "timezone": "Timezone",
-            "version": "Version",
-            "virtualenv": "Virtual Environment"
-        }
+  "system_health": {
+    "info": {
+      "arch": "CPU Architecture",
+      "chassis": "Chassis",
+      "dev": "Development",
+      "docker": "Docker",
+      "docker_version": "Docker",
+      "hasio": "Supervisor",
```

Comment:
Adding it here is not needed, but since we are in beta it's easier to pick this commit containing both instead of adding the lokalise commit that may contain keys we can not use?

Also... prettier made this bigger than what it needed to be...